### ISSUE TITLE
Build integration test

### ISSUE THREAD

**Issue by asudoh** [carbon-design-system/ibm-dotcom-library#2461]
### The problem

Efforts around #2198 and #2436 revealed that there are some kind of issues that doesn't surface in our Storybook environment and surfaces only with running our build in application environment.

### The solution

#2198 keeps track of the effort that creates an environment that emulates certain kind of applications. We can catch the error automatically by integration tests that runs such emulated application environment.

We could use Selenium/Webdriver for this kind of tests, but it's barely suitable for CI integrated to PRs, because they tend to take long time and to sometimes pass and sometimes fail both due to WebDriver nature. ([Ref](https://github.com/w3c/webdriver/blob/master/webdriver-bidi/webdriver.md))

Given errors in build does not have to be multiple-browser tests, we can use Pupeteer for this. We can potentially use [Playwright](https://github.com/microsoft/playwright) anytime we see the need for multiple-browser tests. Pupeteer/Playwright APIs are almost identical.

**Comment by jeffchew** [comment]
Lining up for Sprint 12.

cc: @RobertaJHahn

**Comment by jeffchew** [comment]
@asudoh actually, is the scope of this story the PR we have already merged, or for full coverage of the package?

**Comment by asudoh** [comment]
You are right @jeffchew that #2462 covers the initial work. So I think it's good to close this and queue up any follow-up works as separate issue as they arise.
